livebooks/exercises/A_intro.livemd

```diff
@@ -19,34 +19,47 @@ If you run Livebook locally, get the value of your `PATH` env by running `$ echo
 
 Value returned by `Node.self()` can be used to get the metrics from the running pipelines.
 
-Go to `livebooks/metrics.livemd` go visualize them.
+Go to `livebooks/metrics.livemd` to visualize them.
 
 ```elixir
 Node.self()
 ```
 
 ## Introduction
 
-* Pipeline is an Elixir Process that manages [Membrane](https://hexdocs.pm/membrane_core/readme.html) Elements, which handle the processing of multimedia data.
-* To make a module a Pipeline, you need to add the line `use Membrane.Pipeline` to it.
-* Pipelines can implement Membrane [callbacks](https://membrane.stream/learn/get_started_with_membrane/3#callbacks).
-* Each callback receives as arguments:
-  * The pipeline state, which can be modified (as in GenServer).
-  * The context, which is generated by the framework and contains various information, including about Pipeline's Elements.
-  * Sometimes other arguments, depending on the callback.
-* Each callback must return a two-element tuple `{actions, state}`, where
-  * `actions` is the list of actions the framework will execute ([docs](https://hexdocs.pm/membrane_core/Membrane.Pipeline.Action.html)).
-  * `state` is the (possibly modified) state of the element, which will be passed to the next invocation of any callback.
-* As mentioned, Elements handle the actual data processing.
-* For a module to become an Element, it must `use` `Membrane.Filter`, `Membrane.Source`, `Membrane.Sink` or `Membrane.Endpoint`.
-* Like Pipelines, Elements can implement [callbacks](https://membrane.stream/learn/get_started_with_membrane/3#callbacks), which must return a tuple `{actions, state}`.
-* Elements have output and input `pads`.
-* Elements can be linked with each other using pads: if an element has an output pad, it can be linked with another element that has an input pad.
-* Elements send buffers with data via output pads using actions. When an element sends a buffer via an output pad, the linked element receives this buffer from the respective input pad.
-* Elements can have zero, one, or multiple input and output pads (take a look at the pictures below)
-* Before sending the first data buffer, Elements must send a `stream format` using the `:stream_format` action ([docs](https://hexdocs.pm/membrane_core/Membrane.Element.Action.html#t:stream_format/0)).
-* `stream format` describes what kind of data will be contained in the buffers, e.g., it may specify that each buffer will contain one RGB video frame of dimensions 1920p x 1080p.
-* `stream format` must match the `accepted_format` field in the definition of the pad via which it was sent.
+Pipeline is an Elixir Process that manages [Membrane](https://hexdocs.pm/membrane_core/readme.html) Elements and handles the processing of multimedia data.
```

Comment:
```suggestion
Pipeline is an Elixir process that manages [Membrane](https://hexdocs.pm/membrane_core/readme.html) Elements so that they cooperate to the process multimedia data.
```